play/PlayerRules/CharacterCreation/02_Classes.txt

```diff
@@ -2,645 +2,696 @@ Core classes:
 powerful spells within that branch. See the Tome of Spells for more 
 information.
------------------------
-Name: Warlock
-The Warlock has been trained in the arcane, with specific emphasis on 
-destructive spells for direct combat. Powerful Warlocks are able to turn the 
-tide in a battle, but are more succeptible to damage than their allies.
-Unlocks access to the Warlock spells in the Spell Tome.
------------------------
-Name: Controller
-Controllers hold interest in swaying the battle through affecting their 
-targets with less than lethal means. This doesn''t necesarily mean they are 
-above hurting those they combat. On the contrary, their means often lead to a 
-slower demise for their opponents.
-Unlocks access to the Controller spells in the Spell Tome.
-----------------------
-Name: Tactician
-The Tactician sees the battlefield from a different angle than their more 
-combat oriented fellows. Rather than approach a problem head-on, they prefer 
-to pursue alternate solutions, shaping the battlefield to favor their allies.
-Unlocks access to the Tactician spells in the Spell Tome
-
-"Sorcerer" - If the Mage is the performer, the Sorcerer is the composer. They 
+	-----------------------
+	Name: Warlock
+	The Warlock has been trained in the arcane, with specific emphasis on 
+	destructive spells for direct combat. Powerful Warlocks are able to turn the 
+	tide in a battle, but are more susceptible to damage than their allies.
+	Unlocks access to the Warlock spells in the Spell Tome.
+	-----------------------
+	Name: Controller
+	Controllers hold interest in swaying the battle through affecting their 
+	targets with less than lethal means. This doesn''t necessarily mean they are 
+	above hurting those they combat. On the contrary, their means often lead to a 
+	slower demise for their opponents.
+	Unlocks access to the Controller spells in the Spell Tome.
+	----------------------
+	Name: Tactician
+	The Tactician sees the battlefield from a different angle than their more 
+	combat oriented fellows. Rather than approach a problem head-on, they prefer 
+	to pursue alternate solutions, shaping the battlefield to favor their allies.
+	Unlocks access to the Tactician spells in the Spell Tome
+======================================
+
+
+=====   SORCERER   =====
+If the Mage is the performer, the Sorcerer is the composer. They 
 shroud themselves in secrecy, operating at the highest levels of their 
 organizations. Typically they spend their time researching new uses of Nanites 
 and reverse engineering captured technology. As the designers and researchers, 
 Sorcerers generally don''t need to have a Tome of Spells handy to cast their 
 spells. However, this also means that their spell knowledge is limited to 
 their memory, so they frequently shift around a mental list of spells which 
 can be used.
-Passives: Knowledge Nanites - 10; +50 nanites, +10 per level.
+Class Primary Attribute: Intelligence
```

Comment:
Not having wisdom in our system is unfortunate here. I propose that we do want to distinguish our primary casting classes with primary attribute, and since we don't have Wis to swap sorcerers to, what if we swapped mages to Cha? Unlike Sorcerers who are designing spells, mages get all their spell information literally laid out in front of them. As the description says, performers and composers.